scout/server/blueprints/institutes/templates/overview/cases.html

```diff
@@ -28,8 +28,8 @@
         {{ form.search_type(class="form-control") }}
       </div>
       <div class="col-4">
-        {{ form.search_term.label(class="control-label") }}
-        {{ form.search_term(class="form-control", placeholder='example:18201') }}
+          <label class="control-label" for="search_term">Search cases</label>
+          <input class="form-control" id="search_term" name="search_term" placeholder="example:18201" type="text" pattern="[^<>?!=\\/]*" title="Characters ^<>?!=\/ are not allowed">
```

Comment:
```suggestion
          <input class="form-control" id="search_term" name="search_term" placeholder="example:18201" type="text" pattern="[^<>?!=\\/]*" data-bs-toggle="tooltip" title="Characters ^<>?!=\/ are not allowed">
```

Reply:
I tried locally and your suggestion doesn't display a tooltip. Why a tooltip BTW? This is form validation works fine already:

<img width="442" alt="image" src="https://user-images.githubusercontent.com/28093618/184872612-1424be01-3fec-4f9e-a486-682363766225.png">

Reply:
Ahh OK, title is actually used by the HTML5 validation to display the error!

Reply:
It seems to be rather variable across user agents; see https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern. Oh, one thing, the caret should not be in the title list - it is the NOT for the pattern match operation. 😊